File: app/src/main/java/com/dmi/perfectreader/layout/run/TextRun.java
package com.dmi.perfectreader.layout.run;

import com.dmi.perfectreader.style.FontStyle;

public class TextRun extends Run {
    private final CharSequence text;
    private final FontStyle style;

    public TextRun(String text, FontStyle style) {
        this.text = text;
        this.style = style;
    }

    public CharSequence text() {
        return text;
    }

    public FontStyle style() {
        return style;
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/layout/run/TextRun.kt
package com.dmi.perfectreader.layout.run

import com.dmi.perfectreader.style.FontStyle

class TextRun(text: String, private val style: FontStyle) : Run() {
    private val text: CharSequence

    init {
        this.text = text
    }

    fun text(): CharSequence {
        return text
    }

    fun style(): FontStyle {
        return style
    }
}
